addons/mail/static/src/chatter/web_portal/chatter.js

```diff
@@ -63,27 +58,23 @@ export class Chatter extends Component {
         });
 
         onMounted(() => {
-            this.changeThread(this.props.threadModel, this.props.threadId, this.props.webRecord);
-            if (!this.env.chatter || this.env.chatter?.fetchData) {
-                if (this.env.chatter) {
-                    this.env.chatter.fetchData = false;
-                }
-                this.load(this.state.thread, this.requestList);
-            }
+            this.state.thread = this.store.Thread.insert({
+                model: this.props.threadModel,
+                id: this.props.threadId,
+            });
```

Comment:
just at a glance but threadId can be null and in this case, missing thread.name can be a problem or not?